src/tool/cli.rs

```diff
@@ -61,6 +61,8 @@ impl Cli {
 pub enum Commands {
     #[clap(visible_alias = "gen", short_flag = 'g')]
     Generate(GenerateArgs),
+     /// Copies the code from ??? to your clipboard for pasting on leetcode
```

Comment:
Sorry for confusing you, I meant that we need to document it and the question marks need to be replaced once we decide how we want to handle picking which file to copy from.